function error5_121(message5_121) { }
never;
{
    throw new Error5_121(message5_121);
}
function fail5_121() {
    return error5_121("Something failed");
}
function infiniteLoop5_121() {
    while (true) {
    }
}
